booting_windows/readme.md

```diff
@@ -0,0 +1,76 @@
+# Running Windows over u-root
+
+## Prerequisites
+**WARNING**: the scripts will require sudo priviliges. It is highly advised that
+you read them carefully before running them!
+
+1. Golang, see `https://golang.org/`.
+1. Packages required to build Linux kernel
+1. A functional Windows Server 2019 image, asumed to exist at
+   `"${WORKSPACE}"/win2019.raw`. `setup.sh` will create a masking image over it,
+   so the original image will not be modified.
+1. An environment variable `EFI_WORKSPACE`, where files will be downloaded to or
+   otherwise created.
+
+## Installing the Modified u-root
+1.  Install u-root:
+
+    ```shell
+    go get github.com/u-root/u-root
```

Comment:
I needed to ensure that ${HOME}/go/bin was in my path. E.g.,
export PATH=${HOME}/go/bin:${PATH}

Perhaps add a test that `which u-root` actually finds something?

Reply:
Added to prerequisites instructions in readme.md. See b28937d6